### PR TITLE
Added Sky+HD

### PR DESCRIPTION
**Pull request by GreenTurtwig**
**Description:**
Support for Sky+HD.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28074

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/sky_plus_hd.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ha_release: "0.102"
+---
+
+The `sky_plus_hd` integration allows you to control a [Sky](https://www.sky.com/)+HD box.
+
+To add a Sky+HD box to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
```
Comment (klaasnicolaas) [inline source/_integrations/sky_plus_hd.markdown:17]: Only required config items

File: source/_integrations/sky_plus_hd.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  - platform: sky_plus_hd
+    host: 192.168.0.2
```
Comment (klaasnicolaas) [inline source/_integrations/sky_plus_hd.markdown]:
```suggestion
    host: IP_ADDRESS
```

File: source/_integrations/sky_plus_hd.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Sky+ HD"
+description: "Instructions on how to integrate a Sky+HD box into Home Assistant."
+logo: sky_plus_hd.png
+ha_category:
+  - Media Player
+ha_iot_class: Local Polling
+ha_release: "0.102"
```
Comment (klaasnicolaas) [inline source/_integrations/sky_plus_hd.markdown]:
```suggestion
ha_release: 0.102
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/sky_plus_hd.markdown (hunk 1 above)
Comment (GreenTurtwig) [reply source/_integrations/sky_plus_hd.markdown:17]: Done :+1:

File: source/_integrations/sky_plus_hd.markdown (hunk 2 above)
Comment (GreenTurtwig) [reply source/_integrations/sky_plus_hd.markdown]: Done :+1:

File: source/_integrations/sky_plus_hd.markdown (hunk 3 above)
Comment (GreenTurtwig) [reply source/_integrations/sky_plus_hd.markdown]: Done :+1:

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request closed without merging**